### PR TITLE
nino initial

### PR DESCRIPTION
**Pull request by kyrylloleynikov**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ function isVinnikCool() {
   return true;
 }
 
+function Nino() {
```
Comment (VadimZhiltsov) [inline index.js:5]: Add function to export, to bring this amazing function to the world

**Review by VadimZhiltsov — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -2,6 +2,10 @@ function isVinnikCool() {
...
+  return "Mne ne vsyo ravno";
```
Comment (mbielkin-virtru) [inline index.js]: use singlequotes please

**Comment by kyrylloleynikov** [comment]
@VadimZhiltsov @mbielkin-virtru conflicts and comments resolved

File: index.js (hunk 1 above)
Comment (kyrylloleynikov) [reply index.js:5]: Nice catch, added

**Pull request merged**